Source/WebGPU/WebGPU/QuerySet.mm

```diff
@@ -34,11 +34,12 @@
 
 Ref<QuerySet> Device::createQuerySet(const WGPUQuerySetDescriptor& descriptor)
 {
-    if (descriptor.nextInChain)
+    auto count = descriptor.count;
+    if (descriptor.nextInChain || count > 4096 || !isValid()) {
```

Comment:
4096 comes from the WebGPU specification

Reply:
maybe add a comment or a constant?